canned/consul.go

```diff
@@ -22,8 +23,10 @@ type Consul struct {
 
 func NewConsul(ctx context.Context) (*Consul, error) {
 	os.Setenv("TC_HOST", "localhost")
+	skipReaper, _ := strconv.ParseBool(os.Getenv("TESTCONTAINERS_SKIP_REAPER"))
```

Comment:
keep the variable name as - TESTCONTAINERS_RYUK_DISABLED

Reply:
Okay. Was setting that but then thought the reaper can be something else too (other than ryuk). So kept it same as the  variable skipReaper. 
Will change to TESTCONTAINERS_RYUK_DISABLED

Reply:
done.